applications/API/tests/test_endpoints.py

```
@@ -0,0 +1,209 @@
+        self.assertEqual(response.data["content"], self.pending_insult.content)
+
+    def test_update_own_insult(self):
+        """Test that users can update their own insults"""
+        self.api_client.force_authenticate(user=self.user)
+        url = reverse(
+            "insult-detail", kwargs={"insult_id": self.active_insult.insult_id}
+        )
+        updated_content = "Updated content"
+        response = self.api_client.patch(url, {"content": updated_content})
+
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)